src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write code here
+const employeesBox = document.querySelector('ul');
+const employees = document.querySelectorAll('li');
```

Comment:
You could only create one variable 'employeesBox' and use it like
`[...employeesBox.children]` instead of creating another variable 'employees'